paywall/src/utils/callToAction.ts

```diff
@@ -0,0 +1,23 @@
+import { KeyStatus, PaywallCallToAction } from '../unlockTypes'
+
+// status is assumed to be the "highest" rank key status of all keys
+// available, as determined by getHighestStatus in the keys utils
+// Returns the default call to action if any is missing, because that
+// one is required and will always be present.
+export function getCallToAction(
```

Comment:
This is used to determine which call to action should be shown on the checkout. It's a refactored version of what was previously inside the Checkout component, which bloated the code and made the component hard to read.